version.php

```diff
@@ -24,7 +24,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2021042800;                 // The current module version (Date: YYYYMMDDXX).
+$plugin->version   = 2021101200;                 // The current module version (Date: YYYYMMDDXX).
 $plugin->requires  = 2015111610;  // 3.0         // Requires this Moodle version.
 $plugin->component = 'mod_ubicast';              // Full name of the plugin (used for diagnostics).
 $plugin->release   = '3.8 (Build: 2021042800)';  // Human-readable version name.
```

Comment:
You should also change this version to: `3.9 (Build: 2021101200)`

Reply:
PS: Since the version should change for each validation on moodle.org, please set a different version when you will change this file, for example: `2021112300`.

Reply:
fix in last commit